calculator.rb

```diff
@@ -0,0 +1,63 @@
+#Welcome user and accept numbers to be evaluated. Validate user input.
+puts "Welcome to the calculator program!"
+puts "You will need to enter two numbers to be evaluated."
+puts "Please enter the first."
+begin
+  number_1 = gets.chomp
+  number_1 = Float(number_1)
+  rescue
+    puts "There is an error with your number. Please make sure you're using numbers/decimals only and try again."
+  retry
+end
+
+puts "Please enter another number."
+begin
+  number_2 = gets.chomp
+  number_2 = Float(number_2)
+  rescue
```

Comment:
You've written almost exactly the same code here twice, to get the first number and the second number. Could you DRY that up by putting this logic in a method?